packages/mui-icons-material/custom/Mastodon.js

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import createSvgIcon from './utils/createSvgIcon';
+
+export default createSvgIcon(
+  <path d="M22.457 5.673c-.325-2.44-2.429-4.363-4.923-4.735-.42-.063-2.015-.292-5.708-.292H11.8c-3.694 0-4.487.229-4.908.292-2.424.362-4.638 2.09-5.175 4.559-.259 1.215-.286 2.563-.238 3.8.068 1.773.081 3.543.24 5.31.11 1.172.303 2.336.576 3.482.51 2.116 2.577 3.876 4.603 4.595 2.168.749 4.5.873 6.734.359a9.84 9.84 0 0 0 .729-.201c.542-.175 1.179-.37 1.646-.713a.054.054 0 0 0 .022-.04v-1.712a.05.05 0 0 0-.04-.049.049.049 0 0 0-.021 0 18.475 18.475 0 0 1-4.37.516c-2.533 0-3.214-1.214-3.41-1.72a5.382 5.382 0 0 1-.296-1.356.05.05 0 0 1 .018-.041.048.048 0 0 1 .043-.01c1.408.343 2.851.517 4.3.516.347 0 .695 0 1.043-.01 1.456-.04 2.991-.116 4.424-.399.036-.007.071-.013.102-.022 2.26-.439 4.41-1.816 4.63-5.302.008-.138.028-1.438.028-1.58.001-.485.154-3.434-.022-5.247Zm-3.479 8.7h-2.376V8.49c0-1.239-.511-1.87-1.55-1.87-1.141 0-1.713.747-1.713 2.223v3.22h-2.363v-3.22c0-1.476-.572-2.223-1.714-2.223-1.033 0-1.548.631-1.55 1.87v5.883H5.339V8.311c0-1.238.313-2.222.939-2.952.645-.727 1.492-1.101 2.543-1.101 1.216 0 2.135.473 2.748 1.417l.591 1.003.592-1.003c.613-.944 1.532-1.417 2.747-1.417 1.05 0 1.896.374 2.544 1.101.625.729.938 1.713.938 2.952l-.002 6.062Z" />,
```

Comment:
I grabbed this from the [joinmastodon.org repo](https://github.com/mastodon/joinmastodon/blob/master/public/logos/logo-black.svg)